src/lib.rs

```diff
@@ -158,14 +157,13 @@ impl Game {
                 player.x += self.speed * time_steps_passed
             }
 
-            if !inputs.q && self.last_input.q {
+            if inputs.q {
```

Comment:
I guess this means that if you hold down kill you just kill everyone nearby.

That's fine though, gotta add a kill cooldown anyways